containers/dev/m1-docker-compose.yml

```diff
@@ -3,6 +3,18 @@ services:
 
   elasticsearch5:
     build: ./m1-images/elasticsearch5
+  #elasticsearch7:
+  #  image: elasticsearch:7.17.0
+  #  environment:
+  #    - cluster.name=elasticsearch7
+  #    - discovery.type=single-node
+  #    - xpack.security.enabled=true
+  #    - xpack.security.authc.api_key.enabled=true
+  #    - xpack.security.authc.anonymous.roles=superuser # allows no auth for test purpose
+  #    - ELASTIC_PASSWORD=ductile
+  #  ports:
+  #    - "9207:9200"
+  #    - "9307:9300"
```

Comment:
If this needs to stay here, can you comment why in there? Or remove it altogether if it is actually dead.

Reply:
I don't understand why this was removed in the first place, but I'll figure that out another time.

Reply:
That was removed to keep compose files DRY. With docker compose you can merge multiple compose files together, stacking them into a chain: `docker compose -f docker-compose-1.yml -f docker-compose-2.yml`. In that case, `docker-compose-2.yml` file might contain only configuration options that need to be redefined and additional services not present in `docker-compose-1.yml` file.

The main idea for `m1-docker-compose-file.yml` is to help build images required for `aarch64` architecture (Mac M1/2):
```sh
# Execute this command only once
docker compose -f containers/dev/docker-compose.yml -f containers/dev/m1-docker-compose.yml build
```
After its done, you can start the development environment using the main docker compose file only and forget about `m1-docker-compose.yml`:
```sh
docker compose -f containers/dev/docker-compose.yml up -d
```